manifests/spoke/spoke-work-deployment.yaml

```diff
@@ -0,0 +1,48 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: {{ .SpokeCoreName }}-work-agent
+  namespace: {{ .SpokeCoreNamespace }}
+  labels:
+    app: spoke-work-agent
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: spoke-work-agent
+  template:
+    metadata:
+      labels:
+        app: spoke-work-agent
+    spec:
+      serviceAccountName: {{ .SpokeCoreName }}-sa
+      containers:
+      - name: spoke-agent
+        image: {{ .WorkImage }}
+        imagePullPolicy: IfNotPresent
+        args:
+          - "/work"
+          - "agent"
+          - "--cluster-name={{ .ClusterName }}"
+          - "--kubeconfig=/spoke/hub-kubeconfig"
```

Comment:
I would expect a hub-kubeconfig flag. `--kubeconfig` is normally for self

Reply:
It looks to me like this will change to `--hub-kubeconfig` in https://github.com/open-cluster-management/work/pull/10